test/tests/get-gtest.cmake

```diff
@@ -0,0 +1,65 @@
+# Gets GTest from a remote
+include(ExternalProject)
+
+ExternalProject_Add(
+    googletest-distribution
+    URL https://github.com/google/googletest/archive/release-1.8.0.tar.gz
```

Comment:
Make sure to update the link, as it changed on `develop`: https://github.com/OpenRCT2/OpenRCT2/commit/e01cd0fd40bac48b32a11bad65ee4cad499474cf